discount-details-block/README.md.liquid

```diff
@@ -0,0 +1,5 @@
+# Block Extension
+
+Admin block extensions enable app developers to build custom functionality into the context of Shopify Admin. This discount details block surfaces as a cards in the discount details page using the "admin.discount-details.function-metafields-render" extension block target to enhance the merchant experience. Developers can build the content of this extensions using Shopify's UI Extension components for Discounts.
```

Comment:
Could we make this more specific to discount function metafields, since it's a task specific extension and later on we may allow generic block extensions to be pinned to the discount details page. Possibly rename the folder too if it's possible 🙇